### ISSUE TITLE
Can I use this on Tabbar item?

### ISSUE THREAD

**Issue by ablyBikramjeet** [sunilsharma08/IGCMenu#33]
Hello Sir,

Hope you are doing well!

Thanks a lot for provide wonderful library. I try to implement this library on tabbar item but am unable to do. Can I use this library on tabbar middle item for show and hide. Second Can I change titles of of menus.

Thanks & Best Regards,
Bikramjeet Singh

**Comment by sunilsharma08** [comment]
Hi @ablyBikramjeet 

Currently library does not support `UITabBarItem` but you can do it with some tweaks like keeping a button on top of tabbar item and on that button bind library instance.

Yes, you can change menu title as 
```Swift
igcMenu.menuItemsNameArray = [NSArray arrayWithObjects:@"Home",@"Like",@"Search",@"User",@"Buy",nil];
```
Menu color and icons are also customisable, please see Readme section [create-instance-of-class-igcmenu-and-configure-attributes](https://github.com/sunilsharma08/IGCMenu#create-instance-of-class-igcmenu-and-configure-attributes) for more details.

**Comment by ablyBikramjeet** [comment]
@sunilsharma08 Sir, 

Thanks a lot for your response.

According your suggestion I am able to bind a button on top of tabbar item and on that button bind library instance. But I need to show these buttons upper of tabbar. As like this :- 

![Screenshot 2020-05-18 at 9 42 31 AM](https://user-images.githubusercontent.com/52451314/82173727-eda56f00-98eb-11ea-9ead-1d79bde4601a.png)

As now this library covered all view. And also want dismiss when User touch on disableBackground view. So Please help me.

![Simulator Screen Shot - iPhone6S - 2020-05-18 at 09 37 38](https://user-images.githubusercontent.com/52451314/82173631-94d5d680-98eb-11ea-8ea1-6264c29c7c73.png)


![Simulator Screen Shot - iPhone6S - 2020-05-18 at 09 37 43](https://user-images.githubusercontent.com/52451314/82173640-9b644e00-98eb-11ea-8bbc-e565e85ce9f1.png)

Waiting for your positive response.

Thanks & Best Regards,
Bikramjeet Singh

**Comment by sunilsharma08** [comment]
Hi @ablyBikramjeet 

Again solution is not straight forward.

First you can easily make background transparent by setting property `backgroundType` to `None`.

Then create a empty view on top of products view(the area that you want to disable) and set its color something like `[UIColor whiteColor] colorWithAlphaComponent: 0.7];`

Add tap gesture on this empty view and on tap target close menu `[igcMenu hideCircularMenu];` and hide empty view to make it normal.

In future will try to add these feature in library. These requirements will make it more customisable.
Feel free to raise PR if you want to add some feature.

Thanks

**Comment by ablyBikramjeet** [comment]
Ok Sir, Thanks a lot for your response.

1) Can I move above purple background buttons. Please refer the screenshot. 

![Simulator Screen Shot - iPhone6S - 2020-05-19 at 18 21 19](https://user-images.githubusercontent.com/52451314/82328530-a7d8cb80-99fd-11ea-87f2-77db08eb6f38.png)

2) Can I change the title color and font style of Post, Sell, Review and Arcade labels. Please refer screenshot.

![Simulator Screen Shot - iPhone6S - 2020-05-19 at 18 20 36](https://user-images.githubusercontent.com/52451314/82328766-e4a4c280-99fd-11ea-809d-a62586c0e3ed.png)

Waiting for your positive response.

Thanks & Best Regards,
Bikramjeet Singh

**Comment by sunilsharma08** [comment]
> 1. Can I move above purple background buttons. Please refer the screenshot.

Yes, you can move up or down menu items by changing property `menuRadius` also you can change menu items size by changing property `menuHeight`.

> 2. Can I change the title color and font style of Post, Sell, Review and Arcade labels. Please refer screenshot.

Earlier it was not possible but I have updated library for these customisation. Now you can change title color and font use property `menuTitleColor` and `menuTitleFont` respectively.

But doing so you need to update library, if you're using cocoa pods then run command `pod update IGCMenu` and if your have integrated manually then re-download library.

Thanks
